Tools/LambdaTestTool-v2/src/Amazon.Lambda.TestTool/Services/ApiGatewayRouteConfigService.cs

```diff
@@ -142,15 +142,20 @@ private bool IsRouteConfigValid(ApiGatewayRouteConfig routeConfig)
             return false;
         }
 
-        var occurrences = routeConfig.Path.Split("{proxy+}").Length - 1;
```

Comment:
for validating the path i think you may be able to actually just use the regex that aws uses 

```
^(\{[\w.:-]+\+?\}|[a-zA-Z0-9.:_-]+)$
```

![image](https://github.com/user-attachments/assets/6924b4c8-207f-4821-a06d-822d02bc05f5)

would that work?

Reply:
Done